reference/nats-protocol/nats-protocol/README.md

```diff
@@ -71,19 +71,22 @@ When using the updated client protocol (see [`CONNECT`](./#connect) below), `INF
-* `host`: The IP address used to start the NATS server, by default this will be `0.0.0.0` and can be configured with `-client_advertise host:port`
-* `port`: The port number the NATS server is configured to listen on
-* `max_payload`: Maximum payload size, in bytes, that the server will accept from the client.
-* `proto`: An integer indicating the protocol version of the server. The server version 1.2.0 sets this to `1` to indicate that it supports the "Echo" feature.
-* `client_id`: An optional unsigned integer (64 bits) representing the internal client identifier in the server. This can be used to filter client connections in monitoring, correlate with error logs, etc...
-* `auth_required`: If this is set, then the client should try to authenticate upon connect.
-* `tls_required`: If this is set, then the client must perform the TLS/1.2 handshake. Note, this used to be `ssl_required` and has been updated along with the protocol from SSL to TLS.
-* `tls_verify`: If this is set, the client must provide a valid certificate during the TLS handshake.
-* `connect_urls` : An optional list of server urls that a client can connect to.
-* `ldm`: If the server supports _Lame Duck Mode_ notifications, and the current server has transitioned to lame duck, `ldm` will be set to `true`.
+|name|description|type|required|
+|----|-----------|----|--------|
+|`server_id`|The unique identifier of the NATS server.|string|true|
+|`version`|The unique identifier of the NATS server.|string|true|
+|`go`|The version of golang the NATS server was built with.|string|true|
+|`host`|The IP address used to start the NATS server, by default this will be `0.0.0.0` and can be configured with `-client_advertise host:port`.|string|true|
+|`port`|The port number the NATS server is configured to listen on.|int|true|
+|`max_payload`|Maximum payload size, in bytes, that the server will accept from the client.|int|true|
+|`proto`|An integer indicating the protocol version of the server. The server version 1.2.0 sets this to `1` to indicate that it supports the "Echo" feature.|int|true|
+|`client_id`|The internal client identifier in the server. This can be used to filter client connections in monitoring, correlate with error logs, etc...|uint64|false|
+|`auth_required`|If this is set, then the client should try to authenticate upon connect.|bool|false|
+|`tls_required`|If this is set, then the client must perform the TLS/1.2 handshake. Note, this used to be `ssl_required` and has been updated along with the protocol from SSL to TLS.|bool|false|
+|`tls_verify`|If this is set, the client must provide a valid certificate during the TLS handshake.|bool|false|
+|`connect_urls`|List of server urls that a client can connect to.|[string]|false|
+|`ldm`|If the server supports _Lame Duck Mode_ notifications, and the current server has transitioned to lame duck, `ldm` will be set to `true`.|bool|false|
+|`headers`|Whether the server supports headers|bool|false|
```

Comment:
happy to reformat all of these tables into 'prettier' versions, but would be good to get confirmation on the content before tuning that.